src/web/app/components/beambox/right-panel/LayerPanel.tsx

```diff
@@ -26,28 +30,45 @@ import { isMobile } from 'helpers/system-helper';
 
 import styles from './LayerPanel.module.scss';
 
+import 'react-resizable/css/styles.css';
```

Comment:
I will recommend to move the global styles import to a higher order place (Router.tsx or Beambox.tsx)
or add a _app.scss to include the vendor style sheets if possible